azext_iot/operations/hub.py

```diff
@@ -1866,7 +1889,32 @@ def _iot_c2d_message_receive(target, device_id, lock_timeout=60):
             payload = {"properties": {}}
 
             if "etag" in result.headers:
-                payload["etag"] = result.headers["etag"].strip('"')
+                eTag = result.headers["etag"].strip('"')
+                payload["etag"] = eTag
+
+                if ack:
+                    ack_response = {}
+                    if ack == SettleType.abandon.value:
+                        six.print_("__Abandoning message__")
```

Comment:
Instead of printing for this command, I think these should be logger entries. Also for future reference, you don't have to use `six.print_` since we dropped Py 2.7 support.

Reply:
Ok cool - switched to `logger.debug` - unless you think this should be an `info` statement instead?

Reply:
debug is good